packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -141,7 +147,7 @@ export class BulkApiImpl implements BulkApi {
      * @param response `ok`  if everything is correct
      */
     store = async (request: Request, response: Response): Promise<void> => {
-        requestLogger.info(` * store request received, with body of ${request.headers["content-length"]} bytes`)
+        requestLogger.info(` * store request received, with body of ${request.headers["content-length"]} bytes, params ${JSON.stringify(request.query)}`)
```

Comment:
What if the request is very, very big? Should be cut the string to the first 100 or 1000 characters?

Reply:
I removed the potentially long string entirely from the log.